tools/metrics/import/import-prometheus-s3.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+usage() {
+    # TODO: Optionally being able to provide a different path to save the files to other than default
+    echo "Usage: $0 [--profile AWS_PROFILE] -f FILE_PATH"
+    exit 1
+}
+
+# Initialize variables
+AWS_PROFILE=""
```

Comment:
You're not trying to retrieve an environment variable first ?

Reply:
This is ran locally, unlike the export script. Most people have AWS profiles set up for their CLIs. Environment variables as an option makes sense for the export script because it's ran remotely.

Reply:
The import will be rewritten in the following PR. These import scripts are just there temporarily.